src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -137,8 +141,10 @@ export const ArtworkSidebarBidActionFragmentContainer = createFragmentContainer(
   graphql`
     fragment ArtworkSidebarBidAction_artwork on Artwork {
       myLotStanding(live: true) {
-        active_bid {
-          is_winning
+        most_recent_bid {
```

Comment:
Yea, seems like `active_bid` won't return if it's not winning, whereas `most_recent_bid` always returns...your most recent bid.

Which is correct, right? You'd always need to place a bid that's higher than any of your previous bids, even if they're below the reserve, or some other scenario where they're not winning.